packages/data-flow/src/orchestrator.ts

```diff
@@ -197,6 +209,51 @@ export class Orchestrator {
         });
     }
 
+    private async getMetadataFromEvents(events: AnyIndexerFetchedEvent[]): Promise<string[]> {
+        const ids = new Set<string>();
+
+        for (const event of events) {
+            if ("metadata" in event.params) {
+                ids.add(event.params.metadata[1]);
```

Comment:
```suggestion
                ids.add(event.params.metadata[1]);
```

is metadata[1] safe ?

Reply:
yes is safe to use it this way